robots/accession/remediate_object.rb

```diff
@@ -11,7 +11,7 @@ def initialize
 
         def perform(druid)
           obj = Dor.find(druid)
-          obj.upgrade!
+          obj.upgrade! if obj.respond_to? :upgrade!
```

Comment:
i *think* it's actually ok to just remove this call entirely, and drop the new tests, as i think the `upgrade!` capability was removed from dor-services late last year:
https://github.com/sul-dlss/dor-services/pull/260